src/arrayMethodJoin.js

```diff
@@ -5,7 +5,25 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let finalString = '';
+
+    for (let indexOfElement = 0;
+      indexOfElement < this.length;
+      indexOfElement++) {
```

Comment:
в цикле for принято называть переменную `i`
```suggestion
    for (let i = 0; i < this.length; i++) {
```

Reply:
Ок, значить преборщив)) Після ваших зауважень..